Makefile

```diff
@@ -4,12 +4,20 @@ all: generate format
 
 generate:
 	@mkdir -p ./pb
+	@./merge-proto.sh \
+		fig/tendermint/codec/v1/codec.proto \
+		fig/tendermint/transform/v1/transform.proto \
```

Comment:
`transform.proto` is a separate thing, we shouldn't merge it with `codec.proto`. Check this out: https://github.com/streamingfast/sf-near/tree/develop/pb/sf/near